payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardDetailsSectionElementUI.kt

```diff
@@ -29,43 +30,46 @@ fun CardDetailsSectionElementUI(
     controller: CardDetailsSectionController,
     hiddenIdentifiers: Set<IdentifierSpec>,
     lastTextFieldIdentifier: IdentifierSpec?,
+    modifier: Modifier = Modifier,
 ) {
-    Row(
-        horizontalArrangement = Arrangement.SpaceBetween,
-        verticalAlignment = Alignment.CenterVertically,
-        modifier = Modifier
-            .fillMaxWidth()
-    ) {
-        H6Text(
-            text = stringResource(R.string.stripe_paymentsheet_add_payment_method_card_information),
+    Column(modifier) {
```

Comment:
Why do we need the new nesting here?

Reply:
Since we now allow a `Modifier` to affect this element, we want the `Modifier` to affect the entire element and not just the `Section`.

Reply:
Resolved on Slack!